src/partials/DeleteButton.vue

```diff
@@ -1,14 +1,16 @@
 <!-- ADD_LICENSE_HEADER -->
 <template functional>
 	<span class="pointer delete-button" variant="link" v-on:click="listeners.click">
-		<font-awesome-icon icon="times" fixed-width class="icon" />
+		<font-awesome-icon icon="trash" fixed-width class="icon" />
 	</span>
 </template>
 
 <style lang="scss" scoped>
 .delete-button {
 	padding: 4px 4px;
 	border-radius: 4px;
+	margin-left: 1em;
+	float: right;
```

Comment:
Not sure what this does in this case but I usually try to do layouting in the parent component. If this is correcting layout inside this component, ignore this message.

Reply:
true i will check a better location for those.

Reply:
should be fixed